lib/telemetry.js

```diff
@@ -60,5 +66,28 @@ export const startPingLoop = () => {
     point.tag('arch', arch())
     point.tag('deployment_type', DEPLOYMENT_TYPE)
     writeClient.writePoint(point)
-  }, 10 * 60 * 1000)
+    writeClient.flush().catch(handleFlushError)
+    await timers.setTimeout(10 * 60 * 1000) // 10 minutes
+  }
+}
+
+export const runMachinesLoop = async () => {
+  while (true) {
+    const point = new Point('machine')
```

Comment:
How can we distinguish points submitted by different machines? I don't see any machine-specific identifier in the fields.

Reply:
Hm, why do we need to distinguish points by different machines?

Reply:
That's a good question. How do you envision to query this new data?

I assume you want to write a query like "find me a sum of all RAM sizes of active machines last week". (In other words, find how much RAM is collectively offered by our network.)

Let's say Station A with 4GB of RAM reports a telemetry point every day (Mon, Tue,..., Sun). Station B, with 1GM of RAM, reports only one telemetry point on Wednesday. Station C, with 2GB of RAM, reports two telemetry points on Sat & Sun.

How are you going to calculate the correct answer 8 GB?

We may also want to know the max & min peaks (4 GB on a bad day, 8 GB on a good day) and perhaps the p50 value (a bit more than 4 GB).

Reply:
I would say it depends on what query we want to ask. But I see that I can't harm to add a machine ID, to support more queries. I'll add it 👍

Reply:
0cd2668